### ISSUE TITLE
Replication of lidar-only detection result

### ISSUE THREAD

**Issue by zlenyk** [mit-han-lab/bevfusion#355]
Hi!
I just wanted to confirm if the config posted at: configs/nuscenes/det/transfusion/secfpn/lidar/voxelnet_0p075.yaml should results in the 64.5mAP result. 
We are running it like this:
torchpack dist-run -np 8 python tools/train.py configs/nuscenes/det/transfusion/secfpn/lidar/voxelnet_0p075.yaml --data.samples_per_gpu 1 --optimizer_config.cumulative_iters 4

However we can only get up to 62.5mAP (when measured during training) and 63mAP (when measured during evaluation).
Do you think that changing samples_per_gpu to 1 and optimizer_config.cumulative_iters to 4 could result in such a big change?

Thank you!

**Comment by kentang-mit** [comment]
Hi @zlenyk,

There are two things you probably want to modify. First is to move the box origin from (0.5, 0.5, 0) back to (0.5, 0.5, 0.5) [here](https://github.com/mit-han-lab/bevfusion/blob/db75150717a9462cb60241e36ba28d65f6908607/mmdet3d/datasets/nuscenes_dataset.py#L321). It has some impact on the GT database construction. Second is to avoid using gradient accumulation for backbone training. This is because the detector has a lot of BN layers, and gradient accumulation will not recover the large batch results if the network contains BN layers. Besides that, please remember to rerun `tools/test.py` to get the final accuracy after you finished training. The accuracies reported during training is a bit lower than the correct values.

Best,
Haotian

**Comment by ChiCheng123** [comment]
> Hi @zlenyk,
> 
> There are two things you probably want to modify. First is to move the box origin from (0.5, 0.5, 0) back to (0.5, 0.5, 0.5) [here](https://github.com/mit-han-lab/bevfusion/blob/db75150717a9462cb60241e36ba28d65f6908607/mmdet3d/datasets/nuscenes_dataset.py#L321). It has some impact on the GT database construction. Second is to avoid using gradient accumulation for backbone training. This is because the detector has a lot of BN layers, and gradient accumulation will not recover the large batch results if the network contains BN layers. Besides that, please remember to rerun `tools/test.py` to get the final accuracy after you finished training. The accuracies reported during training is a bit lower than the correct values.
> 
> Best, Haotian

Hi @kentang-mit ,

Why need to move the box origin from (0.5, 0.5, 0) back to (0.5, 0.5, 0.5) in the lidar-only experiment?

Thanks
